org.freedesktop.appstream.generator.yaml

```diff
@@ -0,0 +1,148 @@
+---
+app-id: org.freedesktop.appstream.generator
```

Comment:
not completely sure of the policy to use `org.freedesktop` here.

Reply:
Well, this is a Freedesktop project and I am the upstream maintainer of AppStream, so I am sure it's okay in this case ;-)